code/vendor/botan/src/botan_all.cpp

```diff
@@ -33210,9 +33210,9 @@ void GOST_34_11::add_data(const uint8_t input[], size_t length)
          m_position = 0;
          }
       }
-
-   const size_t full_blocks = length / hash_block_size();
-   const size_t remaining   = length % hash_block_size();
+   const auto hash_block_size_v = hash_block_size();
```

Comment:
This is a third-party library; which is also probably somewhat overdue for updating client-side but newer versions have weird defaults that break ROS proxying so we so far only update it server-side as it's more consequential there.